packages/dashboard/e2e/tests/dashboard/DashboardPage.ts

```diff
@@ -20,9 +21,12 @@ export class DashboardPage {
     this.lineChartWidgetButton = page.getByRole('button', {
       name: 'add Line widget',
     });
-    this.testWidgetButton = page.getByRole('button', {
+    this.textWidgetButton = page.getByRole('button', {
       name: 'add Text widget',
     });
+    this.kpiWidgetButton = page.getByRole('button', {
+      name: 'add KPI widget',
```

Comment:
nit: can use the text from `WidgetSelectorMap` https://github.com/awslabs/iot-app-kit/blob/main/packages/dashboard/e2e/tests/utils/grid.ts#L14